src/test/resources/change_behavior_update.xml

```diff
@@ -32,58 +32,56 @@
 	</data>
 	<data type="boolean">
 		<id>boolean_1</id>
-		<title>Edit behavior separately without transition</title>
+		<title>Edit behavior on transitions</title>
 		<init>false</init>
 		<action trigger="set">
 			bln: f.this,
 			txt0: f.text_0,
-			txt1: f.text_1;
+			trans: t.t1;
```

Comment:
why is `t1` referenced, if it is unused?

Reply:
Unnecessary transition removed.